lib/transform.js

```diff
@@ -52,7 +52,10 @@ module.exports = class Transform {
         const slotMap = new Map([['default', []]]);
         const nodes = treeAdapter.getChildNodes(root);
         nodes.forEach(node => {
-            if (!treeAdapter.isTextNode(node)) {
+            if (
+                !treeAdapter.isTextNode(node) &&
+                !treeAdapter.isCommentNode(node)
```

Comment:
should not be handled in this way.. 

This fix mean would completely skip comment nodes from child templates which is not the expected output.

Reply:
ok, so why do we handle text nodes like?

Tomorrow I will take a look on passing it through.

Reply:
> ok, so why do we handle text nodes like?

yup, that was a bit puzzling, thus was thinking it's ok to have comments handled the same way

Reply:
Totally forgot why i did it that way. Will update once i remember. 

But, with the fix #195, we can even include text nodes(space) as part of child templates.